.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,39 @@
+name: integration-test 
+
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Postgres Test
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_HOST: localhost
+          POSTGRES_USER: root
+          POSTGRES_PASSWORD: secret
+          POSTGRES_DB: optimus_db
+          TEST_OPTIMUS_DB_URL: "postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:5432/$POSTGRES_DB?sslmode=disable"
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:  
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+      - name: Checkout out code into the Go module directory
+        uses: actions/checkout@v2 
+      - name: Test
+        run: make test  
```

Comment:
I don't think this will run tests that are marked with build tags, that is, it won't really run integration tests of Postgres